core-aam/index.html

```diff
@@ -10655,7 +10628,11 @@ <h3>Changes to document content or node visibility</h3>
               </tr>
             </tbody>
           </table>
-          <p>Fire these events for node changes where the node in question is an element and has an <a class="termref">accessible object</a>:</p>
+          <p>
+            Fire the following events for node changes where the node in question is an element and has an <a class="termref">accessible object</a>. The accessibility subtree of a node is its
+            [=accessible object=] in the <a>accessibility tree</a> and its descendants in that tree. It does not include objects which have relationships other than parent-child in that tree. For
```

Comment:
I see that "in that tree" is used more than once; is this redundant?

Reply:
I just spent a moment thinking about removing it, but ultimately I think it's more clear with it in. In fact, to be even more clear, I think we should write: 
```suggestion
            [=accessible object=] in the <a>accessibility tree</a> and its descendants in that tree. It does not include accessible objects which have relationships other than parent-child in that tree. For
```